brave/brave-core
# [Omnibox]: Experiment with including more history results

Partially resolves https://github.com/brave/brave-browser/issues/35063

| Before | After |
|--------|--------|
| ![Screenshot from 2024-01-23 14-40-18](https://github.com/brave/brave-core/assets/7678024/8ba80ff5-e912-4287-a8ce-2ca545a4ab14) | ![Screenshot from 2024-01-23 14-40-43](https://github.com/brave/brave-core/assets/7678024/68a939c4-5983-4155-85bf-ccb8cc4a6a41) |
| ![Screenshot from 2024-01-23 14-40-00](https://github.com/brave/brave-core/assets/7678024/8318959d-0811-4055-814a-df2c2866b42d) | ![Screenshot from 2024-01-23 14-39-35](https://github.com/brave/brave-core/assets/7678024/571f163a-3692-48a7-9dd0-4aa836f4b88d) |
| ![Screenshot from 2024-01-23 14-38-56](https://github.com/brave/brave-core/assets/7678024/8241c40c-7772-47e6-9c2e-f211dac5bebb) | ![Screenshot from 2024-01-23 14-39-25](https://github.com/brave/brave-core/assets/7678024/421619f4-1d97-42bc-8f8b-a7f7c00b3c4b) |






## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. Enable the brave://flags/#brave-history-more-search-results flag
2. Search for a title from history more than a week old
3. You should get results (in the current Brave, you often won't, depending on how many times you've visited the url)
![Screenshot from 2024-01-23 14-40-18](https://github.com/brave/brave-core/assets/7678024/b2afc6f5-c614-408f-80b5-db277bdf62d5)
